README.md

```diff
@@ -167,6 +168,27 @@ expect(descendant).not.toContainElement(ancestor)
 // ...
 ```
 
+### `toContainHtml`
+
+```typescript
+toContainHtml(htmlText: String)
+```
+
+Assert whether a string representing a HTML element is contained in another Element:
+
+```javascript
+// add the custom expect matchers once
+import 'jest-dom/extend-expect'
+
+// ...
+// <span data-testid="parent"><span data-testid="child"></span></span>
+const parent = queryByTestId('parent')
+expect(parentElement).toContainHtml('<span data-testid="child"></span>')
+// ...
+```
+
+> Note: Chances are you probably do not need to use this method. We encourage testing from the perspective of how the user perceives the app in a browser. That's why testing against a specific DOM structure is not advised. It might come in handy in rare cases such as if your users manipulate the DOM such as a content management tool - where users are expected to type custom HTML. please use: [`toContainElement`](#tocontainelement)
+
```

Comment:
I came up with an alternative that perhaps got too lengthy, so feel free to just take ideas from it. Cause I still think the text above is missing some key features:

> Chances are you probably do not need to use this matcher. We encourage testing from the perspective of how the user perceives the app in a browser. That's why testing against a specific DOM structure is not advised.
>
> It could be useful in situations where the code being tested renders html that was obtained from an external source, and you want to validate that that html code was used as intended.
>
> It should not be used to check DOM structure that you control. **A good rule of thumb would be to not use it if you find yourself passing a string literal as the expected html.** In such cases please obtain actual references to the elements you want to check for (using `document.querySelector` or other equivalent methods) and use [`toContainElement`](#tocontainelement) instead.